shared/data/experiments-out.js

```diff
@@ -30,5 +30,20 @@ module.exports = {
                 }
             }
         }
+    },
+    u: {
+        name: 'Facebook click to load experiment',
+        description: 'Test blocking all Facebook and replacing it with clickable elements.',
+        active: true,
+        atbExperiments: {
+            d: {
```

Comment:
Do you need a control group here, or are we just using the experiment as a feature flag?

Reply:
There is a control variant defined in the experiment setup. If a user is in that group, they will still be in the experiment group, but the feature will be disabled. It's not really clear from the code in the extension that it exists though.